docs/project_defn.rst

```diff
@@ -298,6 +298,11 @@ using `GCP Batch <https://cloud.google.com/batch>`_ and `Cloud Run <https://clou
     *  ``vcpus``: Number of CPUs to allocate for running each simulation. Default: 1.
     *  ``memory_mib``: Amount of RAM memory needed for each simulation in MiB. Default: 1024.
        For large multifamily buildings this works better if set to 2048.
+    *  ``boot_disk_mib``: Optional. Extra boot disk size in MiB for each task. This affects how
+       large the boot disk will be (see the `Batch OS environment docs`_ for details) of the
+       machine(s) running simulations (which is the disk used by simulations). This will likely need
+       to be set to at least 2,048 if more than 8 simulations will be run in parallel on the same
+       machine (i.e., when vCPUs per machine_type ÷ vCPUs per sim > 8). Default: None.
```

Comment:
I thought we were discussing the default being 30 GB? Do you mean the default extra?

Reply:
There are two levels of default:

1. The default value here, `None`, which means "Don't make any explicit request to Batch for disk size." By not making such an explicit request...
2. When Batch calculates what the disk size will be, that will end up being 30 GB — at the time of writing!

I want to leverage their default behavior; i.e., I don't want to introduce a redundant default. I also don't want to introduce code or documentation that would be inaccurate if they change their behavior (e.g., if they changed the minimum from 30 GB to 40 GB).